static/js/ui.js

```diff
@@ -27,6 +27,8 @@ exports.get_scroll_element = function (element_selector) {
     var element = element_selector.expectOne()[0];
     if (element.SimpleBar) {
         return $(element.SimpleBar.getScrollElement());
+    } else if ('simplebar' in element.dataset) {
+        return $(new SimpleBar(element).getScrollElement());
```

Comment:
Maybe add a comment explaining what this is for, for posterity?

Reply:
Added.